covid_xprize/examples/prescriptors/neat/README.md

```diff
@@ -19,29 +19,28 @@ that scales quadratically with the number of calls.
 
 ### Setup
 
-These assume the `covid-xprize` repo is located at `<workspace>/covid-xprize`.
+Assuming you have cloned this repository and your working directory is currently the root of the cloned
+repository.
 
 If you have not already done so, install requirements, which include neat-python:
-```
-cd <workspace>/covid-xprize
+```shell script
 pip install -r requirements.txt
 ```
 
-Ensure that `covid-xprize` is in your PYTHONPATH:
-```
-export PYTHONPATH=$PYTHONPATH:<workspace>/covid-xprize/
+Ensure that the repository is in your PYTHONPATH:
+```shell script
+export PYTHONPATH=$PYTHONPATH:$(pwd)
 ```
 
 ### Training Prescriptors
 
 The script `train_prescriptors.py` gives an example of how to train prescriptors.
-Here's how it's used:
-```
-cd <workspace>/covid-xprize/covid_xprize/examples/prescriptors/neat/
-python train_prescriptors.py
+Here's how it's used. Run this from the root of your cloned repository:
+```shell script
+python covid_xprize/examples/prescriptors/neat/train_prescriptor.py
```

Comment:
This was the typo -- the original had `python train_prescriptors.py`, prescriptorS with an S, which doesn't exist.